cn_indian_payroll/cn_indian_payroll/overrides/accual_benefit_claim.py

```
@@ -64,32 +51,20 @@ def employee_accrual_submit(self) :
 
     def calculate_grosspay(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `calculate_grosspay`

Add return type annotation: `None`

(ANN201)